README.md

```diff
@@ -0,0 +1,24 @@
+<p align="center"><img src="https://dydx.exchange/icon.svg?" width="256" /></p>
+
+<h1 align="center">v4 Chain</h1>
+
+<div align="center">
+  <a href='https://github.com/dydxprotocol/v4-chain/blob/main/LICENSE'>
+    <img src='https://img.shields.io/badge/License-BSL_1.1-blue' alt='License' />
+  </a>
+</div>
+
+## Indexer
+The v4 Indexer is a set of microservices which provides a set of rich APIs and websocket channels for a dYdX v4 application.
+
+## Proto
+This directory defines all protos for the v4 Chain.
+
+## Protocol
+Sovereign Blockchain built using Cosmos SDK & CometBFT for v4 of the dYdX Protocol.
+
+## v4-proto-js
+Generated Javascript package for v4 Chain protobufs.
+
+## v4-proto-py
+Generated Python package for v4 Chain protobufs.
```

Comment:
nit: format these all with the file name? (i.e. lowercase, could also include a trailing slash)

Reply:
good call, done!